src/targets/gpu/fuse_ops.cpp

```diff
@@ -310,6 +331,115 @@ void move_standard_front(std::vector<instruction_ref>& args)
         std::swap(*it, args.front());
 }
 
+struct find_gelu
+{
+
+    static auto erf_fn()
+    {
+        return match::name("gpu::erf")(
+            match::arg(0)(match::used_once(),
+                          match::name("gpu::mul")(match::arg(0)(match::any().bind("x")),
+                                                  match::arg(1)(match::has_value(M_SQRT1_2)))));
+    }
+
+    auto matcher() const
+    {
+        return match::name("gpu::mul")(match::arg(0)(match::name("gpu::mul")(
+                                           match::arg(1)(match::args(match::has_value(0.5f))))),
```

Comment:
This should be `either_arg`. There is no guarantee that this will be the second argument.